packages/aragon-client/src/index.js

```diff
@@ -68,6 +68,34 @@ export class AppProxy {
     )
   }
 
+  /**
+   * Resolve an address' identity, using the requested provider.
+   *
+   * @param  {string} address Address to resolve.
+   * @param  {string} providerName Provider to use.
+   * @return {Observable} Single-emission observable that emits the resolved identity
+   */
+  resolveAddressIdentity (address, providerName = 'local') {
+    return this.rpc.sendAndObserveResponse(
+      'address_identity',
+      ['resolve', address, providerName]
+    ).pluck('result')
+  }
+
+  /**
+   * Request an address' identity be modified with the requested provider.
+   *
+   * @param  {string} address Address to modify.
+   * @param  {string} providerName Provider to use.
+   * @return {Observable} Single-emission observable that emits if the modification was successful
+   */
+  requestAddressIdentityModification (address, providerName = 'local') {
```

Comment:
Not actually sure if we need to provide the `providerName` here and in `resolveAddressIdentity()`.

I'm starting to think it may make more sense for clients to be "agnostic" to the identity providers, and for `@aragon/wrapper` to handle the identity resolution stack.

Reply:
I agree. For now I'm removing it from the client calls.